karols-frontend/src/actions/actions.js

```diff
@@ -0,0 +1,29 @@
+export const SELECTED_CHOICESHOP = "CHOSE_CHOICESHOP"
+export const CHOICE_PRESTATION = "CHOICE_PRESTATION"
+export const ADD_SLOTS = "ADD_SLOTS"
+export const CHOOSE_SEX = "CHOOSE_SEX"
+export const CHOOSE_SERVICE = "CHOOSE_SERVICE"
+
+export const makeChoicePrestation = prestationId => ({
```

Comment:
Be consistent with function names choose to start with make or not but it must be all the same
start the name with make and finish the name with action has the benefit to make all developers sure this is an action creator function

Reply:
line 1 is not correct